sql/moz-fx-data-shared-prod/firefox_desktop_derived/desktop_installs_v1/query.sql

```diff
@@ -3,7 +3,7 @@
             -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
 WITH download_token_info AS (
   SELECT
-    dltoken,
+    dltoken AS attribution_dltoken,
```

Comment:
IIRC column renames require a DDL query since they can't be done by API, which `bqetl_artifact_deployment` would do. So in addition to this PR, you might need to run something like:
```
ALTER TABLE moz-fx-data-shared-prod.firefox_desktop_derived.desktop_installs_v1
  RENAME COLUMN dltoken TO attribution_dltoken
  ...
;
```
on the production table.